eox_nelp/settings/common.py

```diff
@@ -3,15 +3,15 @@
 """
 from __future__ import absolute_import, unicode_literals
 
-from importlib.util import find_spec
-
 SECRET_KEY = 'a-not-to-be-trusted-secret-key'
-INSTALLED_APPS = (
+INSTALLED_APPS = [
+    'django.contrib.admin',
     'django.contrib.auth',
     'django.contrib.contenttypes',
     'django.contrib.sessions',
-)
-EOX_AUDIT_MODEL_APP = 'eox_audit_model.apps.EoxAuditModelConfig'
+    'eox_nelp',
```

Comment:
don't we need `eox-theming` ?

Reply:
I set it in the past, but I  test without it, and works. Mainly it would be that mfe_config_view is not tested complete.
![image](https://user-images.githubusercontent.com/51926076/220957241-b709de9e-e254-46cf-80f2-c26d9fa0449b.png)
Furthermore, the use of eox-theming is not like a django app is more like a library like bs4 because we only use a method from it.
https://github.com/eduNEXT/eox-nelp/blob/master/eox_nelp/mfe_config_api/v1/views.py#L7
should we add it?
also eox-tenant is requirement but we don't add it as installed app.

Reply:
no, we shouldn't